updating/index.adoc

```diff
@@ -0,0 +1,72 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. Platform administrators are automatically notified when an update is available for their cluster. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+* xref:../updating/understanding-the-update-service.adoc#understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public. You can check the following information:
+
+** xref:../updating/understanding-the-update-service.html#unmanaged-operators_understanding-the-update-service[Understanding support policy for unmanaged Operators]
+
+* xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]:  For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information. You can check the following actions:
+
+** xref:../updating/installing-update-service.adoc#update-service-install[Installing the OpenShift Update Service Operator]
+** xref:../updating/installing-update-service.adoc#update-service-delete-service[Uninstalling the OpenShift Update Service Operator]
+** xref:../updating/installing-update-service.adoc#update-service-create-service[Creating an OpenShift Update Service application]
+** xref:../updating/installing-update-service.adoc#update-service-uninstall[Deleting an OpenShift Update Service application]
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are tied to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. If you do not want the Cluster Version Operator to fetch available updates from the upgrade recommendation service, you can use the `oc adm upgrade channel` command in the `oc` to configure an empty channel. Check out the following information:
```

Comment:
It looks like you are using "upgrade" and "update" interchangeably here. I believe that "update" is preferred. This might be difficult to correct due to the legacy use of the term "upgrade". Your call on how/if to fix.

Use "connected" rather than "tied"?

"See the following information about..." instead of "Check out the..."

Reply:
Addressed